bridges/GBAtempBridge.php

```diff
@@ -76,70 +84,56 @@ public function collectData(){
 		case 'N':
 			foreach($html->find('li[class=news_item full]') as $newsItem) {
 				$url = self::URI . $newsItem->find('a', 0)->href;
-				$time = intval(
-					$this->extractFromDelimiters(
-						$newsItem->find('abbr.DateTime', 0)->outertext,
-						'data-time="',
-						'"'
-					)
-				);
+				$img = $this->getURI() . $newsItem->find('img', 0)->src . '#.image';
+				$time = $this->findItemDate($newsItem);
 				$author = $newsItem->find('a.username', 0)->plaintext;
 				$title = $newsItem->find('a', 1)->plaintext;
 				$content = $this->fetchPostContent($url, self::URI);
-				$this->items[] = $this->buildItem($url, $title, $author, $time, $content);
+				$this->items[] = $this->buildItem($url, $title, $author, $time, $img, $content);
+				unset($newsItem);
```

Comment:
This should not be necessary

Reply:
When testing the bridge, I had memory exhaustion issues using default values from my PHP instance, and manually unsetting the item helped resolving that because I think otherwise the item stays in memory for a bit before garbage collection occurs. I'll add a comment in source code about this problem.